src/ServiceProvider.php

```diff
@@ -29,7 +29,11 @@ public function register(): void
                 throw ApiKeyIsMissing::create();
             }
 
-            return OpenAI::client($apiKey, $organization);
+            return OpenAI::factory()
+                ->withApiKey($apiKey)
+                ->withOrganization($organization)
+                ->withHttpClient(new \GuzzleHttp\Client(['timeout' => config('openai.request_timeout', 30)]))
```

Comment:
This is a breaking change?

Reply:
Yes, this is possibly a breaking change:
- If currently the requests are taking longer than 30 seconds
- If there are other HTTP clients installed, it's currently maybe not using the Guzzle Client (I think this one is unlikely and even if so, it should not change the behaviour)

But anyway I suggest to create a new major release.